main/src/globals.h

```diff
@@ -63,13 +65,23 @@ extern bool debuggingOn;
 extern bool stopwatchOn;
 extern bool outputSerialOn;
 extern bool outputSDOn;
-extern int designOrCalibrate;
-extern int acceptCal;
-extern int designPreset;
-extern int pauseSelection;
+extern char selectedDesignPreset;
+
+// Persistent settings
+#define SETTINGS_ADDRESS 128 // EEPROM Address for settings
+extern Settings settings;
+
+// gCode parameters
+extern bool drawGCode;
 
-// Material properties
-extern float matThickness;
+// Preset parameters
+extern float radius;
+extern float length;
+extern float width;
+extern float deepth;
```

Comment:
```suggestion
extern float depth;
```
Fix variable name typo